src/renderer/widgets/MatrixModal/ui/Credentials.tsx

```diff
@@ -16,7 +12,6 @@ const Credentials = ({ onLogOut }: Props) => {
     toggleProgress();
     try {
       await matrix.logout();
-      onLogOut();
```

Comment:
Now after `logout` we will be redirected to `LoginModal`
Instead of weird modal closure